tools/GSDumpGUI/Forms/frmMain.cs

```diff
@@ -151,6 +206,31 @@ private void ReloadGsdxDlls()
 
             Settings.GSDXDir = gsdxFolder.FullName;
             _internalLogger.Information("Completed GSdx Loading Procedures");
+
+            string[] paths = { "", "\\plugins", "\\dll", "\\dlls" };
+
+            foreach (FileSystemWatcher w in _dllWatcher)
+            {
+                w.EnableRaisingEvents = false;
+                w.Dispose();
+            }
+
+            _dllWatcher.Clear();
+
+            for (int i = 0; i < 4; i++)
+            {
+                try
+                {
+                    FileSystemWatcher w = new FileSystemWatcher(Settings.GSDXDir + paths[i], "*.dll");
+                    //w.Changed += OnDllDirChange;
+                    w.Created += OnDllDirChange;
+                    w.Deleted += OnDllDirChange;
+                    w.Renamed += OnDllDirChange;
+                    w.EnableRaisingEvents = true;
+                    _dllWatcher.Add(w);
+                }
+                catch { }
```

Comment:
I think there should be a logger available here. Why not logging errors?

Reply:
I thought it might just spam a few "path does not exists" every time.

Reply:
Ah I missed that. In that case you’d typically first check for FileExists. Exceptions are a bad program flow (in the net world) and you should avoid them as much as possible. Although I don’t see how it could really introduce issues here.